pyriemann/embedding.py

```diff
@@ -1,8 +1,14 @@
 """Embedding covariance matrices via manifold learning techniques."""
 
 import numpy as np
-from sklearn.base import BaseEstimator
+from scipy.linalg import solve
+from scipy.sparse import csr_matrix
+
+from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.manifold import spectral_embedding
+from sklearn.manifold._locally_linear import null_space
```

Comment:
do not import private functions from sklearn. THere is no backward compat guarantee to these. It can break pyriemann at any time